tests/phpunit/elementor/core/common/modules/connect/apps/test-common-app.php

```diff
@@ -529,4 +529,39 @@ private function get_connected_user() {
 
 		return $user;
 	}
+
+	public function test_ge111t_remote_authorize_url() {
+		// Arrange
+		$_GET['utm_source'] = 'test-source';
+		$_GET['utm_medium'] = 'test-medium';
+		$_GET['utm_campaign'] = 'test-campaign';
+		$_GET['utm_not_allowed_param'] = 'test-test';
+
+		$utm_campaign = [
+			'source' => 'transient-source',
+			'medium' => 'transient-medium',
+			'campaign' => 'transient-campaign',
+		];
+
+		set_transient( 'elementor_core_campaign', $utm_campaign );
```

Comment:
Maybe add a const for `elementor_core_campaign`?

Reply:
No needed. It's based on the 3rd party.